memory-viz/bin/cli.js

```diff
@@ -7,26 +7,31 @@ const { program } = require("commander");
 const { json } = require("node:stream/consumers");
 
 function parseFilePath(input) {
-    if (input) {
-        return pathExists(input, `File`);
-    } else {
-        return undefined;
+    const fullPath = path.resolve(process.cwd(), input);
+    if (fs.existsSync(fullPath, "File")) {
```

Comment:
The input path can really be anything and might not even be syntactically valid. Do we want to do a syntax check before trying to resolve the path?

We can use a regex like this: https://stackoverflow.com/questions/73595422/how-do-i-know-if-a-given-path-is-valid-regardless-of-the-existence-or-absence-o